### ISSUE TITLE
(syncthing) Old files/folders left behind on package upgrade

### ISSUE THREAD

**Issue by opusforlife2** [pauby/ChocoPackages#134]
### Chocolatey Version

1.1.0

### Package Version

Not version-specific

### Current Behaviour

Hey. In the `syncthing > tools` folder, there are folders from previous versions left behind, such as:

```
syncthing-windows-amd64-v1.18.3
syncthing-windows-amd64-v1.18.4
syncthing-windows-amd64-v1.18.5
syncthing-windows-amd64-v1.18.6
```

and so on.

In the `syncthing` folder (one level up from `tools`), there are also these old text files:

```
syncthing-windows-amd64-v1.18.3_x64.zip.txt
syncthing-windows-amd64-v1.18.4_x64.zip.txt
syncthing-windows-amd64-v1.18.5_x64.zip.txt
```

and so on.

### Expected Behaviour

Is it possible to ensure these are deleted when the package is upgraded? They took up ~150MB of space before I noticed them.

### Steps To Reproduce

Upgrade the package.

### Environment

```markdown
N/A
```

### Chocolatey Log

```shell
N/A
```

**Comment by opusforlife2** [comment]
BTW: I selected `None` for the Chocolatey license, but that is shown as `No response` here. Maybe that should be changed?

**Comment by pauby** [comment]
>  In the syncthing > tools folder, there are folders from previous versions left behind,

I've added code to remove these.

> In the syncthing folder (one level up from tools), there are also these old text files:

Those belong to Chocolatey so they need to be left alone.

I also deleted the embedded Zip files from the package folder, as once the package is installed they are no longer needed. They are ~ 9MB each.

This was fixed [here](https://github.com/pauby/ChocoPackages/commit/86186635234eaa1f6bc779ddae8b9e3a6a9f7273) and will be in the next release.
